docker-compose.yml

```diff
@@ -12,11 +12,11 @@ services:
             PGSQL_DB_1_POSTGIS: "true"
         volumes:
             # Use a volume to enable database persistence across restarts
-            - /tmp/nodewatcher-db:/var/lib/postgresql/9.5/main
+            - ~/nodewatcher-db:/var/lib/postgresql/9.5/main
```

Comment:
Why this change? Because it is reset between reboots of host laptop? But I do not like that now developers would have a random directory in home directory. One option could be to make this [configurable through variable substitution](https://docs.docker.com/compose/compose-file/#variable-substitution). And leave `/tmp` as default.